.github/workflows/docsite-build-deploy.yml

```diff
@@ -13,7 +13,7 @@ env:
 jobs:
   build-and-deploy:
     concurrency: ci-${{ github.ref }}
-    runs-on: single-cell-8c64g-runner
+    runs-on: sc-dev-64g-runner
```

Comment:
This may also be do-able on a smaller machine, but I think okay to leave for now since we were already using a big one.